webapp/src/main/java/com/cmds/webapp/models/Robot.java

```diff
@@ -17,33 +17,41 @@
 @NoArgsConstructor
 public class Robot {
```

Comment:
This class needs functions for removingTrips and things like that. UpdateStatus needs to be slightly smarter because I will send you things like NEW, IN_PROGRESS, COMPLETED, STUCK. So if get something like COMPLETED you need to remove the trip from the robot. Following this same logic your GET_TRIPS function should only return trips that have a "NEW" status. The flow is as follows:
make a delivery: set its status to NEW
once I pick it up I send a bunch of "IN_PROGRESS: passed x"
and once I'm done ill send "COMPLETED"
So the model only gives me new deliveries and once it gets COMPLETED it'll remove them

Reply:
Done. Let me know if I'm missing any functionality here, will leave this comment open